Core/Core.php

```diff
@@ -371,7 +371,7 @@ function get_parent_class ($object = null) {}
 /**
  * Checks if the class method exists
  * @link https://php.net/manual/en/function.method-exists.php
- * @param mixed $object <p>
+ * @param object|string $object <p>
```

Comment:
Looks like the `mixed` type is actually the correct one – [implementation just returns false](https://github.com/php/php-src/blob/master/Zend/zend_builtin_functions.c#L1124) for any other types. Although it doesn't make much sense to pass anything other than `object|string`, it would be more correct to follow the specification here

Reply:
I see. All right, reverting then.